actions.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Canonical
```

Comment:
copyright year should be changed to 2023, or possibly 2024, in all files (ones that are not part of relation libraries)